ReactApp/car_rental_react/src/components/HomePage/HomePage.js

```diff
@@ -31,167 +29,197 @@ const homepageData = {
     returnLocation: "Return Location",
     returnDate: "Return Date",
     returnTime: "Return Time",
-    experience: "Experience it now.",
-    follow: "Follow us on our social media",
-    copyright: "All Rights Reserved. © xxx",
     enterLocation1: "Enter Location",
-};
+  };
 
   const [branch, setBranch] = useState([]);
+  const [pickUpLocation, setPickUpLocation] = useState("");
+  const [dropOffLocation, setDropOffLocation] = useState("");
+  const [pickUpDate, setPickUpDate] = useState("");
+  const [dropOffDate, setDropOffDate] = useState("");
+  const [pickupTime, setPickupTime] = useState("");
+  const [returnTime, setReturnTime] = useState("");
+
+  //navigate and handleSearch button
+  const navigate = useNavigate();
+
+  const handleSearch = () => {
+    let dateNow = new Date();
+    console.log(dateNow.getTime())
+    if (pickUpLocation == "" || dropOffLocation == "" || pickUpDate == "" || dropOffDate == "") {

```

Comment:
In JS, be careful when you compare. Please use triple equal sign "===" instead of 2 "==".